### PR TITLE
Place workload and iohandler modules under common/sgx

### PR DESCRIPTION
**Pull request by manju956**
common/sgx_workload and common/sgx_iohandler are moved under common/sgx folder.
Also, moved CMakeVariables.txt file to common/sgx which is be used by above two modules.

Signed-off-by: manju956 <manjunath.a.c@intel.com>

### CODE REVIEW

File: common/sgx/workload/CMakeLists.txt [hunk 1]
```diff
@@ -30,7 +30,7 @@ SET(GENERIC_CXX_FLAGS ${DEBUG_FLAGS} "-Wall" "-fPIC" "-Wno-write-strings" "-std=
 
 INCLUDE_DIRECTORIES(${TCF_TOP_DIR}/tcs/core/tcs_trusted_worker_manager/enclave)
 
-SET(WORKLOAD_STATIC_NAME sgx_workload)
+SET(WORKLOAD_STATIC_NAME workload)
```
Comment (manojgop) [inline common/sgx/workload/CMakeLists.txt]: Could you rename the static library as common_sgx_workload. workload -> common_sgx_workload

File: common/sgx/iohandler/CMakeLists.txt [hunk 2]
```diff
@@ -25,7 +25,7 @@ SET(GENERIC_PRIVATE_INCLUDE_DIRS "." "${TCF_TOP_DIR}/tc/sgx/common"
 	"${TCF_TOP_DIR}/tc/sgx/trusted_worker_manager/enclave")
 SET(GENERIC_CXX_FLAGS ${DEBUG_FLAGS} "-Wall" "-fPIC" "-Wno-write-strings" "-std=c++11")
 
-SET(IOHANDLER_STATIC_NAME sgx_iohandler)
+SET(IOHANDLER_STATIC_NAME iohandler)
```
Comment (manojgop) [inline common/sgx/iohandler/CMakeLists.txt]: Could you rename the static library as common_sgx_iohandler. iohandler-> common_sgx_iohandler

File: common/sgx/workload/CMakeLists.txt (hunk 1 above)
Comment (manju956) [reply common/sgx/workload/CMakeLists.txt]: Done

File: common/sgx/iohandler/CMakeLists.txt (hunk 2 above)
Comment (manju956) [reply common/sgx/iohandler/CMakeLists.txt]: Done

**Pull request merged**